app.py

```diff
@@ -1,37 +1,62 @@
 from flask import Flask, jsonify
 import requests
+import functools
 import os
 
 
 app = Flask(__name__)
 
+
 BASE_URL = os.environ.get('BASE_URL',
                           'https://cplab1.techservices.illinois.edu:443/api')
 CLIENT_ID = os.environ.get('CLIENT_ID')
 CLIENT_SECRET = os.environ.get('CLIENT_SECRET')
 
 
+def handle_exception(func):
+    """
+    Handles exceptions and logging for any decorated
+    function.
+    """
+    @functools.wraps(func)
+    def wrapper(*args, ** kwargs):
+        try:
+            func(*args, **kwargs)
+        except Exception:
+            # TODO: Log exception ({func.__name__} threw exception {e}, etc...)
```

Comment:
I like to call this the @#$%! TODO. Rather not merge this with a silent suppress here. Can we at log a message or not catch?

Reply:
Logging to stderr now. The default behavior in a docker container is for stderr to write to a json logfile on the local system. While the container is running the log can be accessed with the ```$ docker logs <CONTAINER ID>``` command. This can be changed to a different driver when going to prod or when the code is on a separate test system so that we can persist/view logs.